lang/ko.js

```diff
@@ -7,19 +7,19 @@ export default {
 	"components.calendar.show": "{month} 표시",
 	"components.dialog.close": "이 대화 상자 닫기",
 	"components.input-date.clear": "지우기",
-	"components.input-date.openInstructions": "",
-	"components.input-date.setToToday": "",
+	"components.input-date.openInstructions": "{Format} 날짜 형식을 사용하십시오. 미니 달력에 접근하려면 아래쪽 화살표를 누르거나 Enter 키를 누르십시오.",
```

Comment:
Need to fix casing on `{format}`
```suggestion
	"components.input-date.openInstructions": "{format} 날짜 형식을 사용하십시오. 미니 달력에 접근하려면 아래쪽 화살표를 누르거나 Enter 키를 누르십시오.",
```